proxy/Main.cc

```diff
@@ -164,10 +164,7 @@ static int cmd_line_dprintf_level = 0;  // default debug output level from ink_d
 static int poll_timeout           = -1; // No value set.
 static int cmd_disable_freelist   = 0;
 
-static int shutdown_timeout = 0;
-
-static volatile bool sigusr1_received = false;
-static volatile bool sigusr2_received = false;
+static volatile bool signal_received[32];
```

Comment:
I'd have to agree with @danobi - it's just a static array, probably worth the extra space to have a portable symbolic constant.